web/util.py

```diff
@@ -24,3 +25,12 @@ def flash_redirect(path: str, message: str | None = None, category="info"):
         flash(message, category)
 
     return redirect(path)
+
+
+def upload_file_to_printer(app, file):
+    user_name = request.headers.get("User-Agent", "ankerctl").split(url_for('app_root'))[0]
```

Comment:
That was in your original function, I didn't add it so /shrug :D

Reply:
Okay, I had to fire up git blame :stuck_out_tongue: 

We're both to blame. You did a search-replace in commit 1969a29fb3e8099b614bc43c0ae533f24121a4e2, and I didn't catch it in the review.

By sheer coincidence, this happens to work, because `url_for('app_root')` returns `/` in "normal" circumstances for this program.

Here's the relevant part of that diff:

```
-    user_name = request.headers.get("User-Agent", "ankerctl").split("/")[0]
+    user_name = request.headers.get("User-Agent", "ankerctl").split(url_for('app_root'))[0]
```

(it's an unrelated problem, so we don't have to fix it here, but feel free)

Reply:
Haha love git blame :D